tests/workers/test_weave_scorer.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
```

Comment:
I think tests for this change are sorely lacking. It would be great to have a test suite that actually tests the end to end functionality: create a monitor, log calls, query and get their scores. I think this primarily comes down to getting a queue into CI